supporting_content/dimensions/pricingmodel.md

```diff
@@ -0,0 +1,33 @@
+# Column: PricingModel - WIP
+
+## Example provider mappings
+
+Current column mappings found in available data sets:
+
+| Provider | Data set                 | Column                    |  Example Values  |
+|----------|--------------------------|---------------------------|------------------|
+| AWS | CUR                      |  product/PurchaseOption | On-Demand, Reserved Instances, Spot Instances, Dedicated Hosts |
+| Google Cloud | BigQuery Billing Export |              |  |
+| Microsoft | Cost Details             | PricingModel             | OnDemand, Spot, Reservation, SavingsPlan |
+
+## Example usage scenarios
+
+Current values observed in billing data for various scenarios:
+
+| Provider | Data set                 | PricingModel     |      |
+|----------|--------------------------|----------------------------|------------------------------------------|
+| AWS | CUR ()                   |                 |              |
+| AWS | CUR ()                   |                |                    |
+| Google Cloud | BigQuery Billing Export |         |                            |
+| Google Cloud | BigQuery Billing Export |         |                               |
```

Comment:
Should this be populated with values you've listed above as example values?

Reply:
It SHOULD BE use, but it COULD NOT be due to time. I will do it now